platform/linux-generic/odp_parse.c

```diff
@@ -40,16 +40,16 @@ uint16_t _odp_parse_eth(packet_parser_t *prs, const uint8_t **parseptr,
 		input_flags.jumbo = 1;
 
 	/* Handle Ethernet broadcast/multicast addresses */
-	macaddr0 = odp_be_to_cpu_16(*((const uint16_t *)(const void *)eth));
+	macaddr0 = odp_be_to_cpu_16(*((const odp_una_u16_t *)(const void *)eth));
```

Comment:
Is there some benefit in the intemediate casting to (const void *)?

Reply:
I don't think so. Removed in v3.